### ISSUE TITLE
Detekcia viacerých chýb a návrhy so spodobovaním

### ISSUE THREAD

**Issue by misog** [sk-spell/hunspell-sk#46]
V issue  #42 sa vyriešil problém, že systém nezobrazoval návrhy so správnou diakritikou ak slovo obsahovalo dve chyby, napríklad slovo `motyl`.

V závere issue sa rieši spodobovanie `h` a `ch`.

Spodobovanie je naďalej problematické a preto vznikol tento nový issue. Napríklad, slovo `fčela` dostane správny návrh `včela` ale slovo `fcela` už nedostane návrh `včela`.

Je možné do MAP zakomponovať aj dvojice pre spodobovanie?

Zdroj - sposobovanie
https://www.viemeposlovensky.sk/book/cvicenia-spodobovanie

**Comment by zdenop** [comment]
Problém `fcela` nie je spodobovanie ale 2 chyby vedľa seba.

**Comment by misog** [comment]
Áno, akuálne sa nedetekujú dve chyby ak jedna z nich je chyba pre spodobovanie (fcela). Je to obdobné ako v prípade #42 kde sa neukazovali návrhy pri dvoch chybách ak obe chyby boli v diakritike (motyl).
